server/templates/rich_search.html

```diff
@@ -0,0 +1,38 @@
+{#
+  Copyright 2022 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+#}
+
+{% extends 'base.html' %}
+
+{% set is_hide_full_footer = true %}
+{% set subpage_title = "Rich Search" %}
+{% set subpage_url = url_for('search.rich_search') %}
+{% set title = "Rich Search" %}
+
+{% block head %}
+  <link rel="stylesheet" href={{url_for('static', filename='css/rich_search.min.css')}} >
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Note that you might not need these, and if so, only one of them? Depending on the UI of the page.